### PR TITLE
add link to allow user easily report issue

### PR DESCRIPTION
**Pull request by rmgogogo**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/3030)

### CODE REVIEW

**Comment by rmgogogo** [comment]
https://github.com/kubeflow/pipelines/issues/3024

File: frontend/src/components/SideNav.tsx [hunk 1]
```diff
@@ -548,6 +548,18 @@ export default class SideNav extends React.Component<SideNavProps, SideNavState>
               </div>
             </Tooltip>
           )}
+          <Tooltip
+              title="Report an Issue"
+              enterDelay={300}
+              placement={'top-start'}>
+            <div className={css.envMetadata}>
+              <a href="https://github.com/kubeflow/pipelines/issues/new?template=BUG_REPORT.md"
```
Comment (Bobgy) [inline frontend/src/components/SideNav.tsx]: Please add `rel="noopener"` in the `<a>`. It helps fix a security risk.

Ref: https://mathiasbynens.github.io/rel-noopener/

Comment (Bobgy) [reply frontend/src/components/SideNav.tsx]: probably also, the above two links

**Comment by Bobgy** [comment]
Fixes #3024

**Comment by Bobgy** [comment]
and take a look at https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/kubeflow_pipelines/3030/kubeflow-pipeline-frontend-test/1226809426036396035#1:build-log.txt%3A83

**Comment by Bobgy** [comment]
FYI, you need to merge master to pass travis tests: https://github.com/kubeflow/pipelines/pull/3035

**Comment by rmgogogo** [comment]
> and take a look at https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/kubeflow_pipelines/3030/kubeflow-pipeline-frontend-test/1226809426036396035#1:build-log.txt%3A83

not sure whether you also config the style or because didn't change the file. I configured the Prettier and it has more changes than I expected.

**Comment by Bobgy** [comment]
@rmgogogo It seems you also enabled vscode organize imports feature. We haven't enforced organize imports, so you are seeing more changes. No worry, these changes are good.

Note frontend tests are still failing because some snapshots need to be updated, you can do so by
`npm test -- -u` and run all tests.
https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/kubeflow_pipelines/3030/kubeflow-pipeline-frontend-test/1227486969991270400#1:build-log.txt%3A1838

**Pull request merged**